scripts/header

```diff
@@ -4,7 +4,7 @@ set -e
 cat <<END
 //  Ramda v$(node -p 'require("envvar").string("VERSION", require("./package.json").version)')
 //  https://github.com/ramda/ramda
-//  (c) 2013-$(git show -s --format=%ai | cut -d - -f 1) Scott Sauyet and Michael Hurley
+//  (c) 2013-$(git show -s --format=%ai | cut -d - -f 1) Scott Sauyet, Michael Hurley, and David Chambers
```

Comment:
Never even heard of it before. But as long as it is intentional....
And I noticed that it wasn't used before. Perhaps this will be a breaking change then? ;)